### PR TITLE
Feat:Filtering methods for services and ordering, updated the docs

### PR DESCRIPTION
**Pull request by santorar**
Added different methods for filtering for the services and updated all the Swagger docs changing the language to English.

### CODE REVIEW

File: controllers/userController.go [hunk 1]
```diff
@@ -57,16 +63,15 @@ func Login(c *fiber.Ctx) error {
...
 }
 
-// Profile Obtiene la información del usuario
+// Profile Get the profile of a user
 //
-//	@Summary		Obtiene la información del usuario
-//	@Description	Obtiene la información del usuario de la sesión actual
-//	@Tags			Cuentas
+//	@Summary		Get the profile of a user
+//	@Description	Get the profile of a user
```
Comment (TatoNaranjo) [inline controllers/userController.go:69]: Beware with text identation/format

File: docs/swagger.json [hunk 2]
```diff
@@ -13,42 +13,42 @@
     "paths": {
         "/Login": {
             "post": {
-                "description": "Inicia sesión con un usuario en especifico",
+                "description": "Login a user",
```
Comment (TatoNaranjo) [inline docs/swagger.json:16]: Change this text to -> "Login with an specific user account"

File: docs/swagger.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
     "swagger": "2.0",
     "info": {
-        "description": "Esta es una api para la pagina nightingCare",
+        "description": "This is the API for the NightingCare project",
```
Comment (TatoNaranjo) [inline docs/swagger.json:4]: Change this text to -> "This is an API for Nightingcare Project"

File: docs/swagger.json [hunk 4]
```diff
@@ -13,42 +13,42 @@
...
                 ],
                 "produces": [
                     "application/json"
                 ],
                 "tags": [
-                    "Cuentas"
+                    "Accounts"
                 ],
-                "summary": "Inicia sesión",
+                "summary": "Login a user",
```
Comment (TatoNaranjo) [inline docs/swagger.json:26]: Change this text to -> "Login"

File: docs/swagger.json [hunk 5]
```diff
@@ -13,42 +13,42 @@
...
                 "parameters": [
                     {
                         "type": "string",
-                        "description": "Nombre de usuario",
+                        "description": "User name",
```
Comment (TatoNaranjo) [inline docs/swagger.json:30]: Change this text to -> "Username"

File: docs/swagger.json [hunk 6]
```diff
@@ -58,35 +58,26 @@
         },
         "/Profile": {
             "get": {
-                "description": "Obtiene la información del usuario de la sesión actual",
+                "description": "Get the profile of a user",
```
Comment (TatoNaranjo) [inline docs/swagger.json:61]: Change this text to -> "Get the profile of the logged user"

File: docs/swagger.json [hunk 7]
```diff
@@ -96,37 +87,77 @@
...
+                        "description": "Type of the user",
+                        "name": "type",
+                        "in": "formData",
+                        "required": true
                     }
                 ],
                 "responses": {
                     "201": {
-                        "description": "Registro completado",
+                        "description": "User registered",
```
Comment (TatoNaranjo) [inline docs/swagger.json:154]: Change this text to -> "Successful Registration"

File: docs/swagger.json [hunk 8]
```diff
@@ -136,35 +167,35 @@
         },
         "/service": {
             "get": {
-                "description": "Obtiene un servicio por su id",
+                "description": "Gets a service",
```
Comment (TatoNaranjo) [inline docs/swagger.json:170]: Change this text to -> "Gets a service by id"

File: docs/swagger.json [hunk 9]
```diff
@@ -136,35 +167,35 @@
...
                     "200": {
-                        "description": "Servicio",
+                        "description": "Service",
                         "schema": {
                             "$ref": "#/definitions/servicesdtos.ServiceDto"
                         }
                     },
                     "404": {
-                        "description": "No se encontró el servicio",
+                        "description": "Not Found",
```
Comment (TatoNaranjo) [inline docs/swagger.json:198]: Change this text to -> "Service not found"

File: docs/swagger.json [hunk 10]
```diff
@@ -174,23 +205,70 @@
...
                     "200": {
-                        "description": "Servicios",
+                        "description": "Services",
                         "schema": {
                             "$ref": "#/definitions/servicesdtos.ServicesDto"
                         }
                     },
                     "404": {
-                        "description": "No se encontraron servicios",
+                        "description": "Not Found",
```
Comment (TatoNaranjo) [inline docs/swagger.json:271]: Change this text to -> "No services found"

File: docs/swagger.yaml
Comment (TatoNaranjo) [inline docs/swagger.yaml:1]: Make the same grammatical changes of the Swagger.Json file.

File: docs/swagger.json
Comment (TatoNaranjo) [inline docs/swagger.json:1]: There were a couple of grammatical errors, and some of the text was not clear. Remember that we are doing documentation and we must describe every single thing about the API operation in depth.

File: main.go [hunk 11]
```diff
@@ -24,7 +24,7 @@ func init() {
 
 // @title			NightingCare API
 // @version		1.0
-// @description	Esta es una api para la pagina nightingCare
+// @description	This is the API for the NightingCare project
```
Comment (TatoNaranjo) [inline main.go:27]: Change this text to -> "This is an API for Nightingcare Project"

**Review by TatoNaranjo — APPROVED** [review]
There are a couple of fixes to make, all of which lie in the documentation of the API functions. Because they are minor changes that do not affect the functionality of the code, I will merge them to the main branch instead of sending them for review again. 

However, consider making the changes for the next pull request, or this time I will be forced to send them for review.

File: docs/swagger.json (hunk 6 above)
Comment (santorar) [reply docs/swagger.json:61]: this endpoint retrive the profile for logged and another users

**Pull request merged**